lmms_eval/models/simple/gpt4o_audio.py

```
@@ -0,0 +1,414 @@
+        return self._rank
+
+    @property
+    def world_size(self):
+        return self._world_size
+
+    def generate_until_multi_round(self, requests) -> List[str]:
+        raise NotImplementedError("GPT4O-Audio not support")
+
+    def loglikelihood(self, requests: List[Instance]) -> List[Tuple[float, bool]]:
+        # TODO
+        assert False, "GPT4O-Audio not support"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)